### DISCUSSION TITLE
Docker activity without corresponding GitHub updates

### DISCUSSION THREAD

**Discussion by sheerlox** [docker-library/postgres#1145] [General]
Hello,

I am using pinned digests with Renovate. I received a few update notifications in the last few days and was wondering where the activity was coming from.

My first reaction was to check the GitHub repository, but I do not see commit activity on this timeframe, and thus am wondering what is the process pushing these new digests to the Docker Hub.

Could someone clarify that for me?

Thanks in advance!

**Comment by ImreSamu** [comment]
> I received a few update notifications in the last few days and was wondering where the activity was coming from.

(As a user)
From what I observe, it seems that the base Debian image was updated, 
which likely triggered the rebuilding of the dependent PostgreSQL Debian images. 
I believe this is also important from a security standpoint.

https://github.com/docker-library/official-images/blob/master/library/debian ( updated 4 days ago )

> am wondering what is the process pushing these new digests to the Docker Hub.

I'm not sure if you've read these, but they might be useful for others:
* https://github.com/docker-library/faq#an-images-source-changed-in-git-now-what
* https://github.com/docker-library/faq#how-are-images-built-especially-multiarch

**Comment by sheerlox** [comment]
> From what I observe, it seems that the base Debian image was updated,
which likely triggered the rebuilding of the dependent PostgreSQL Debian images.

That makes total sense! Thanks for linking to the FAQ repository as well, it was very insightful :detective: 

If that's directly related to security updates of the underlying images, I guess I'll configure Renovate to branch auto-merge official Docker digest updates in the background as long as tests pass.

**Reply by ImreSamu** [reply]
> If that's directly related to security updates of the underlying images,

Relevant quote from a member of the Docker Library Postgres repository : 
* _"We update all Debian based images to include any updates in apt packages at least monthly (we regenerate the base images and then rebuild all dependent images)."_  ( via https://github.com/docker-library/postgres/issues/995#issuecomment-1244000332 )

And Bookworm [updated ](https://github.com/docker-library/official-images/commit/bc9070c6b145c707a27d9e5c7db792139e36c2b6) to `bookworm-20231120`
```diff
# bookworm -- Debian 12.2 Released 07 October 2023
- Tags: bookworm, bookworm-20231030, 12.2, 12, latest
+ Tags: bookworm, bookworm-20231120, 12.2, 12, latest
```
